src/lib/CMakeLists.txt

```diff
@@ -583,5 +589,6 @@ endif()
 
 # Configure the regular hyrise library used for tests/server/playground...
 add_library(hyrise STATIC ${SOURCES})
-target_link_libraries(hyrise ${LIBRARIES})
+target_link_libraries(hyrise ${LIBRARIES} -rdynamic)
```

Comment:
I think we should doc why we set compile flags. Otherwise no one knows why they are there and no one will dare to touch them.

Reply:
Right!